### PR TITLE
CNV RN: Single RHV VM import

### PR DESCRIPTION
**Pull request by apinnick**
https://issues.redhat.com/browse/CNV-5371

### CODE REVIEW

**Review by pkliczewski — APPROVED** [review]
Looks good to me

**Comment by vikram-redhat** [comment]
@openshift/team-aligned-docs PTAL.

File: virt/virt-2-4-release-notes.adoc [hunk 1]
```diff
@@ -28,6 +28,8 @@ Pod-based services.
 ** Add, edit, and view virtual CD-ROM devices attached to a virtual machine.
 ** Add and view network attachment definitions with a graphical editor.
 
+* You can import a single Red Hat Virtualization virtual machine with the virtual machine wizard or the CLI.
```
Comment (ousleyp) [inline virt/virt-2-4-release-notes.adoc:31]: s/with/by using

**Review by ousleyp — COMMENTED** [review]
One IBM style nit, but otherwise LGTM

File: virt/virt-2-4-release-notes.adoc (hunk 1 above)
Comment (apinnick) [reply virt/virt-2-4-release-notes.adoc]: Fixed

**Pull request closed without merging**